### PR TITLE
Document editing the system purpose of multiple hosts

### PR DESCRIPTION
**Pull request by spetrosi**
Bug 1901920 - [RFE] Document setting system purpose on multiple hosts
via bulk action

https://bugzilla.redhat.com/show_bug.cgi?id=1901920

### CODE REVIEW

**Comment by suyogsainkar** [comment]
thanks @spetrosi looking good to me; merging.

File: guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc [hunk 1]
```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
...
-== Editing the System Purpose of a Host
+== Editing the System Purpose of Hosts
 
-You can edit the system purpose attributes for a Red{nbsp}Hat Enterprise{nbsp}Linux 8 host. System purpose attributes define which subscriptions to attach automatically to this host. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
+You can edit the system purpose attributes of Red{nbsp}Hat Enterprise{nbsp}Linux 8 hosts. System purpose attributes define which subscriptions to attach automatically to hosts. For more information about system purpose, see https://access.redhat.com/documentation/en-us/red_hat_enterprise_linux/8/html/performing_a_standard_rhel_installation/graphical-installation_graphical-installation#configuring-system-purpose-standard_configuring-system-settings[Configuring system purpose] in the _Performing a standard RHEL installation_ guide.
```
Comment (pondrejk) [inline guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:13]: I don't think this is limited to RHEL8, afaik you can set syspurpose also for RHEL7

File: guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc [hunk 2]
```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
 
 ifeval::["{build}" == "satellite"]
 [id="editing-the-system-purpose-of-a-host"]
-== Editing the System Purpose of a Host
+== Editing the System Purpose of Hosts
```
Comment (pondrejk) [inline guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:11]: Overall I'm not sure why do we replace the procedure for editing a single host, it is still valid, sometimes even  preferable. Unlike the batch edit process, when editing syspurpose attributes in content host details we are able to see what we are changing (i.e. the original values of the attributes)

File: guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc [hunk 3]
```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
...
 
-. In the {Project} web UI, navigate to *Hosts* > *Content Hosts* and click the name of the Red{nbsp}Hat Enterprise Linux{nbsp}8 host that you want to edit.
-. In the *System Purpose* area, click the *Edit* or *Remove* icon for the system purpose attributes that you want to edit, add, or remove.
-. Click *Save*.
-. Click the *Subscriptions* tab and select *Subscriptions*.
-. Click *Run Auto-Attach* to attach subscriptions to your host automatically.
-. Refresh the page to verify that the subscriptions list contains the correct subscriptions.
+. In the {Project} web UI, navigate to *Hosts* > *Content Hosts* and select Red{nbsp}Hat Enterprise Linux{nbsp}8 hosts that you want to edit.
+. Click the *Select Action* button and from the drop-down menu, click *Manage System Purpose*.
+. Select the system purpose attributes that you want to assign to the selected hosts and click *Assign*.
```
Comment (pondrejk) [inline guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:19]: You can assign attributes but there are also *No Change* and *None (Clear)* options that are worth explaining: no change leaves the original value for the attribute in place -- useful when we want to set some attribute for a large number of hosts but don't want to mess with other attributes, none wipes out the attributes

File: guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc (hunk 2 above)
Comment (pondrejk) [reply guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:11]: Maybe also worth saying that hosts need to be registered via the subscription manager, In the single host procedure, if the sub-man status of the host is not ok, you are not even able to proceed to the host details to change syspurpose, but with bulk procedure nothing prevents you from trying to update even the non-ok hosts, obviously that results in errors

**Comment by pondrejk** [comment]
@spetrosi leaving some notes post-merge

File: guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc [hunk 4]
```diff
@@ -8,18 +8,18 @@ include::proc_creating-a-host-in-satellite.adoc[leveloffset=+1]
...
+. Navigate to *Hosts* > *Content Hosts* and select the same Red{nbsp}Hat Enterprise Linux{nbsp}8 hosts to automatically attach subscriptions based on the system purpose.
+. Click the *Select Action* button and from the drop-down menu, click *Manage Subscriptions*.
+. Click *Auto-Attach* to attach subscriptions to all selected hosts automatically.
```
Comment (pondrejk) [inline guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:23]: maybe worth mentioning how to verify the bulk action succeeded from the actual hosts (in fact there was a bug precisely about this https://bugzilla.redhat.com/show_bug.cgi?id=1905979), commands to verify:

subscription-manager service-level
subscription-manager usage
subscription-manager role

Comment (spetrosi) [reply guides/doc-Managing_Hosts/topics/Administering_Hosts.adoc:23]: It is problematic to log in to all hosts, and ideally the procedure should just work, so I will kepp this verification to users.

**Review by spetrosi — COMMENTED** [review]
Hi @pondrejk, thanks a lot for your feedback. I have applied it in the PR #372, please review.

**Pull request merged**